.travis.yml

```diff
@@ -11,6 +11,8 @@ before_install:
   # Needed to deploy pull request and releases
   - sudo apt-get -y install python-pip python-dev
   - pip install awscli --upgrade --user
+  # use production url for price estimator on master or tagged
+  - export PRICE_ESTIMATOR_URL=$([[ $TRAVIS_BRANCH = "master" || $TRAVIS_TAG != "" ]] && echo "production" || echo "develop")
```

Comment:
Sorry it was not in the description. But can we use production one ALSO in the `release/*` and `hotfix/*` branches?

We want those ones AS CLOSE AS POSSIBLE as production